test/e2e/storage/testsuites/volumemode.go

```diff
@@ -275,9 +275,8 @@ func (t *volumeModeTestSuite) DefineTests(driver storageframework.TestDriver, pa
 					"involvedObject.namespace": l.ns.Name,
 					"reason":                   volevents.ProvisioningFailed,
 				}.AsSelector().String()
-				msg := "does not support block volume provisioning"
 
-				err = e2eevents.WaitTimeoutForEvent(l.cs, l.ns.Name, eventSelector, msg, f.Timeouts.ClaimProvision)
+				err = e2eevents.WaitTimeoutForEvent(l.cs, l.ns.Name, eventSelector, "", f.Timeouts.ClaimProvision)
```

Comment:
Please add a comment that we can't check the provisioning error message, because it depends on a CSI driver.

Reply:
+1